.github/workflows/pytest.yml

```diff
@@ -198,11 +198,24 @@ jobs:
           path: coverage
       - name: List downloaded files
         run: |
-          ls -R
+          find coverage -type f
       - name: Upload test coverage report to CodeCov
         uses: codecov/codecov-action@v4
         with:
           directory: coverage
+      - name: Install Micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          init-shell: bash
+          environment-name: coverage
+          create-args: >-
+            python=3.11
+            coverage>=7.4.1
+      - name: Combine coverage data and check that we have required coverage
+        run: |
+          micromamba run -n coverage coverage combine coverage/*/.coverage
+          micromamba run -n coverage coverage xml
+          micromamba run -n coverage coverage report --fail-under=100
```

Comment:
This should implement the simple fixed threshold check.  I want to test it without submitting to the merge queue, but the `workflow_dispatch` trigger isn't working.  Is there something obvious wrong with the conditional @jdangerx? I'm having trouble finding examples to compare against.